### PR TITLE
new title bar

### PR DESCRIPTION
**Pull request by mb324-unikent**
A slightly redesigned title bar. A bit taller to allow the page name and path. Potentially could also show the published status, when that's available.

### CODE REVIEW

File: resources/assets/js/store/modules/site.js [hunk 1]
```diff
@@ -12,7 +12,9 @@ const state = {
 	pageModal: {
 		visible: false,
 		parentId: null
-	}
+	},
+	sitePath: 'site',
+	siteDomain: 'https://webtools-test.kent.ac.uk/'
```
Comment (sy238-unikent) [inline resources/assets/js/store/modules/site.js]: What are these for? sitePath and siteDomain?

We should not be hard-coding Kent-specific stuff like this.

We should also start documenting (even if a couple of inline comments) what any properties we are adding are for.

File: resources/assets/js/store/modules/page.js [hunk 2]
```diff
@@ -26,7 +26,8 @@ const state = {
 	scale: .4,
 	loaded: false,
 	dragging: false,
-	currentSavedState: ''
+	currentSavedState: '',
+	isPublished: true
```
Comment (sy238-unikent) [inline resources/assets/js/store/modules/page.js]: What is this isPublished?

File: resources/assets/js/store/modules/site.js (hunk 1 above)
Comment (mb324-unikent) [reply resources/assets/js/store/modules/site.js]: @sy238-unikent can we add these to the api so we can pull them out for each page/site?

Comment (mb324-unikent) [reply resources/assets/js/store/modules/site.js]: - added comments to the state variables
- behaviour is a bit broken till we can get the frontend domain and site page. This isn't just in the title bar, but also means we can't link people to the correct url to view the page.

File: resources/assets/js/store/modules/page.js (hunk 2 above)
Comment (mb324-unikent) [reply resources/assets/js/store/modules/page.js]: @sy238-unikent take a look at the latest version for comments

**Comment by mb324-unikent** [comment]
note that I've also added in functionality to display the correct page title even when you type a url in the browser directly rather than clicking on a page in the sidebar. The means we don't get the page title as 'Home page' even if you're viewing a different page.

**Comment by sy238-unikent** [comment]
I've made some changes to the way this component interacts with the page state.
I have replaced the properties set on the state which duplicated information that should be in the pageData with getters and updated the component to use these instead.

I also removed the some mutators and properties which were introducing duplication of data into the store.

**Comment by cfc7-unikent** [comment]
We've lost the ability to change page settings.

**Comment by sy238-unikent** [comment]
@cfc7-unikent https://github.com/unikent/astro/pull/88 ?

**Pull request closed without merging**